frontends/web/src/routes/device/bitbox01/settings/components/randomnumber.jsx

```diff
@@ -55,27 +55,22 @@ class RandomNumber extends Component {
         <SettingsButton onClick={this.getRandomNumber}>
           {t('random.button')}
         </SettingsButton>
-        {
-          // @ts-ignore Object is possibly 'undefined'.
```

Comment:
looks like TS doesn't complain about this anymore, also the next `// @ts-ignore` seems to be not needed anymore.